earth_enterprise/src/server/mod_fdb/fdb_reader_manager.cpp

```diff
@@ -19,6 +19,8 @@
 #include <http_log.h>
 #include "common/serverdb/serverdbReader.h"
 
+APLOG_USE_MODULE(fdb);
```

Comment:
This fails to compile on [Travis-CI](https://travis-ci.org/google/earthenterprise/jobs/542729182). Should it be within ifdef (ex. #ifdef APLOG_USE_MODULE) or moved to within Constructor ?